tnyandoro/tic-tac-toe
# Milestone 4: ReadMe Game Instructions

### What does this do? 
 ReadMe Game instructions
### Description of Task to be completed? 
 Contains all information about Tic Tac Toe and how to play it,
### How should this be manually tested? 
*  Click [Here](https://github.com/tnyandoro/tic-tac-toe/tree/readme_game_instructions
), to check out the ReadMe.
